arch-installer.sh

```diff
@@ -127,6 +127,11 @@ trap '
 # clean up after ourselves no matter how we die.
 trap 'exit 1;' SIGINT
 
+if [[ ! -e "$1" ]] && [[ "$1" == /dev/* ]]; then
+    echo "Couldn't find block device $1" >&2
+    exit 1
+fi
+
```

Comment:
I think it would be better to tidy up this check with the checks bellow. I think it makes more sense to have something like:
if [[ !  -b "$1" ]] && [[ "$1" == /dev/* ]]; then
    echo "Couldn't find block device $1" >&2
    exit 1
fi